inji-verify-sdk/README.md

```
@@ -114,14 +140,25 @@ If summariseResults=false, then response should be
             "expiryCheck": { "valid": true },
             "statusChecks": [
-                { "purpose": "revocation", "valid": true, "error": null },
-                { "purpose": "suspension", "valid": true, "error": null },
+                { "purpose": "revocation", "valid": true, "error": null }
             ],
             "claims": {..}
         }
     ]
 }  
 ```
+### Response Fields Summary
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)